metrics/api/src/main/java/module-info.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+module io.helidon.metrics.api {
+
+    requires java.logging;
+
+    requires io.helidon.common.serviceloader;
+    requires io.helidon.config;
+
+    requires microprofile.metrics.api;
```

Comment:
this dependency should be transitive (as it is part of its public API)

Reply:
fixed